<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRecorder</class>
 <widget class="QDialog" name="AddRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a recorder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="idHLayout">
     <item>
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>Recorder identifier:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="idLineEdit">
       <property name="text">
        <string>Rec1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="methodHLayout">
     <item>
      <widget class="QLabel" name="methodLabel">
       <property name="text">
        <string>Select recording method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="methodComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRecorder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
